packages/editor-ui/src/views/NodeView.vue

```diff
@@ -166,7 +166,7 @@ import { useExternalHooks } from '@/composables/useExternalHooks';
 import { useClipboard } from '@/composables/useClipboard';
 import { usePinnedData } from '@/composables/usePinnedData';
 import { useSourceControlStore } from '@/stores/sourceControl.store';
-import { useDeviceSupport } from 'n8n-design-system';
+import { useDeviceSupport } from '@n8n/frontend-composables';
```

Comment:
Do we want to export everything from the root of the directory in a barrel file? I know vite probably is smart enough to tree shake everything, but having each composable as a separate export instead would probably reduce the amount code each of our tools need to parse. Basically we would need to define `exports` in the `package.json` for each composable. See [this](https://github.com/frehner/modern-guide-to-packaging-js-library)

Reply:
As discussed, removed barrel file.